src/v2/controllers/FormController.js

```diff
@@ -17,13 +17,12 @@ import { getV1ClassName } from '../ion/ViewClassNamesFactory';
 export default Controller.extend({
   className: 'form-controller',
 
-  initialize: function () {
-    Controller.prototype.initialize.call(this);
-
-    this.listenTo(this.options.appState, 'change:currentFormName', this.render);
-    this.listenTo(this.options.appState, 'invokeAction', this.invokeAction);
-    this.listenTo(this.options.appState, 'switchForm', this.switchForm);
-    this.listenTo(this.options.appState, 'saveForm', this.handleFormSave);
+  appStateEvents: {
```

Comment:
nice, I wasn't aware of this `appStateEvents`. Where is this prop being exposed ?

Reply:
we can do anything like `${propName}Events`. most of times are `modelEvents`, `collectionEvent`, `stateEvents` where `model`/`collection`/`state` are part of `this` context.
https://github.com/okta/okta-ui/blob/master/packages/courage/src/framework/View.js#L37-L60